tests/browser/ckeditor.jsx

```diff
@@ -278,6 +276,54 @@ describe( 'CKEditor Component', () => {
 				} );
 			} );
 		} );
+
+		it( 'uses the freshest data - component mounted, editor not loaded', () => {
+			const initialData = '<p>Initial data</p>';
+			const changedData = '<p>Changed data</p>';
+			const component = createEditor( {
+				data: initialData
+			} );
+
+			// Component is already mounted, editor is not loaded, `data` prop changes
+			component.setProps( {
+				data: changedData
+			} );
+
+			return waitForEditor( component, 'loaded' ).then( editor => {
+				sandbox.spy( editor, 'setData' );
+
+				return waitForEditor( component );
+			} ).then( editor => {
+				expect( editor.getData().trim() ).to.equal( changedData );
+			} );
+		} );
+
+		it( 'uses the freshest data on mount - component mounted, editor loaded', () => {
+			const initialData = '<p>Initial data</p>';
+			const changedData = '<p>Changed data</p>';
+			const changedDataNext = '<p>Changed data next</p>';
+			const component = createEditor( {
+				data: initialData
+			} );
+
+			// Component is already mounted, editor is not loaded, `data` prop changes
```

Comment:
```suggestion
			// Component is already mounted, editor is not loaded, `data` prop changes.
```

[Code style reference](https://github.com/ckeditor/ckeditor4/blob/major/dev/docs/codestyle.md#210-comments)

Reply:
Done!